src/main/java/frc/robot/commands/SpinKickerCommand.java

```diff
@@ -11,11 +11,11 @@ public SpinKickerCommand(){
     }
```

Comment:
We should rename this class to something that matches what it actually does now. Maybe something like FlyWheelCommand.

Reply:
changed it to "flywheelcommand"